src/components/order-service/order-service.controller.ts

```diff
@@ -418,7 +418,11 @@ export class OrderServiceController implements angular.IController {
       var readyCondition: any = _.find(conditions, {type: "Ready"});
 
       this.ctrl.orderComplete = readyCondition && readyCondition.status === 'True';
-      this.ctrl.error = _.find(conditions, {type: "ProvisionFailed"});
+      this.ctrl.error = _.find(conditions, function(condition: any) {
+        return condition.reason === "ProvisionFailed" ||
+               condition.reason === "ProvisionCallFailed" ||
+               condition.reason === 'ErrorCallingProvision';
+      });
```

Comment:
@pmorie Provisioning a service is failing with:

> "Provision call failed. Error provisioning Instance "whee/django-psql-persistent-4l3nw" of ServiceClass "django-psql-persistent" at Broker "template-broker": Service instance with same id but different attributes exists"

But the condition is still:
type: 'Ready'
status: 'False'
reason: 'ProvisionCallFailed'

Should the type got to 'Failed'?